tests/benchmarks/test_parquet.py

```diff
@@ -0,0 +1,109 @@
+"""
+Parquet-related benchmarks.
+"""
+import uuid
+
+import dask.dataframe as dd
+import dask.datasets
+import distributed
+import fsspec
+import pandas
+import pytest
+from coiled.v2 import Cluster
+
+
+@pytest.fixture(scope="module")
+def parquet_cluster():
+    with Cluster(
+        f"parquet-{uuid.uuid4().hex[:8]}",
+        n_workers=15,
+        worker_vm_types=["m5.xlarge"],
+        scheduler_vm_types=["m5.xlarge"],
+    ) as cluster:
+        yield cluster
```

Comment:
I have a module-scoped cluster that's a bit bigger than the default, and I wanted to keep these well-scoped so they don't tread on other tests running in parallel.